downstream/modules/eda/proc-eda-view-activation-output.adoc

```diff
@@ -0,0 +1,13 @@
+[id="eda-view-activations-output"]
+
+= Viewing activation output
+
+You can view the output of the activations in the *History* tab.
+
+.Procedure
+. Select the *History* tab to access the list of all the times the activation has been executed.
+. Then select the activation in question, this will show you the *Output* produced by that specific execution.
```

Comment:
If we proceed with the above suggestion this should be "...select the activation instance..."

Reply:
Done :+1: